api/.idea/workspace.xml

```diff
@@ -1,36 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's up with this file?

Reply:
Not sure where it came from, but we don't need nested intellij configs afaik